pkg/servicecentral/types.go

```diff
@@ -30,11 +30,36 @@ type ServiceData struct {
 	Stateless            bool   `json:"stateless,omitempty"`
 	BusinessUnit         string `json:"business_unit,omitempty"`
 
+	Attributes []ServiceAttribute
+
 	// Compliance is a read-only field. It can be nil, in which case it means
 	// they have not completed their compliance questions yet
 	Compliance *ServiceComplianceConf `json:"compliance,omitempty"`
 }
 
+type ServiceAttribute struct {
+	Team string
+}
+
+type ServiceAttributeResponse struct {
+	ID      int `json:"id"`
+	Service struct {
```

Comment:
Please don't use anonymous structs like that. Just to be consistent with the rest of the repo.